js/badges.js

```diff
@@ -25,7 +25,16 @@ function badges() {
         var data = "<li id='badgesItem_" + obj.id + "' class='collection-badges'>" +
             "<img src='" + imageDirectory + obj.image + "' alt='" + obj.name + "' title='" + obj.name + "' class='badgesImg'>" +
             "</li>";
-        $("#badgesDisplay").append(data);
+            $("#badgesDisplay").append(data);
+            $("#badge_count").html("(" + count() + "/4)");
+    }
+
+
+    function count() {
+        if( typeof count.counter == 'undefined' ) {
+            count.counter = 0;
```

Comment:
How does this refer to https://github.com/fossasia/labyrinth/pull/721/files#diff-836ce988ddaf69630fa37d1807ee2abbR4 ?